### DISCUSSION TITLE
How to customize Wh. of .F. item set

### DISCUSSION THREAD

**Discussion by Wotan8887** [bananarama92/MBS#6] [Q&A] [answered]
Hello there,

I would like to ask for help. I have installed addon for custom outfits for Wheel of Fortune. It is working nice, but I just can't find any way how to set my own item sets. I have open window, where I can see and write name of my set, locks, clothing strip/ equip, etc. But I just can't find the way, how to set the items which I wish to have in the item set... I tried clicking on my chracter, which is on left, tried put items on it and then go to wheel's setting, tried wrote items names to "outfit code" etc. and nothing. :3 

I have to write some outfit codes to see it on my character on left, how she will looks with it (and then will set to the item set)? Or I just add some items on character (I don't know how to do it :3 ) and then the outfit code is generated itself? Or some other way? May I ask for help please? :) Thank you very much for any help! :)

https://github.com/bananarama92/MBS/commit/46731a9a396c9b50b8960c216ca1d257344bc66f

![image_2023-03-18_21-03-36](https://user-images.githubusercontent.com/128256754/226135414-277be99e-e46e-4ad7-ae42-23035ada87d3.png)

**Comment by bananarama92** [comment, marked as answer]
Hi Wotan, all clothes and restraints are specified via the "outfit code" field, which accepts outfit codes as generated via [WCE](https://wce-docs.vercel.app/) (via the `/exportlooks` command) or [BCX](https://jomshir98.github.io/bondage-club-extended/) (via the "export outfit" button in the wardrobe). Both approaches will produce an outfit code that can subsequently be copy/pasted into the respective field in MBS.

EDIT: Replaced [FBC](https://sidiousious.gitlab.io/bce/) references with [WCE](https://wce-docs.vercel.app/) due to the former being unmaintained and incompatible with current BC versions

**Reply by Wotan8887** [reply]
Got it, it works for me now. Thank you very much for the help! ^^
